CMakeSettings.json

```diff
@@ -2,97 +2,97 @@
   // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
   "configurations": [
     {
-      "name": "x64-Release",
-      "generator": "Visual Studio 15 2017 Win64",
+      "name": "x86-RelWithDebInfo",
+      "generator": "Visual Studio 15 2017",
       "configurationType": "RelWithDebInfo",
-      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "inheritEnvironments": [ "msvc_x64_x86" ],
```

Comment:
I think this environment is invalid. I would simply go for `msvc_x86`.

https://docs.microsoft.com/en-us/cpp/build/cmakesettings-reference?view=vs-2019#environments

Reply:
Thanks, I have changed it to `msvc_x86`.  I got the architecture name from https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=vs-2017, but you are right, the environment names are backwards.  This did work, but I don't know how or why. :)  I think VS may have simply ignored it, and it did an x86 build because of the generator setting.